build/win/BUILD.gn

```diff
@@ -40,10 +42,12 @@ action("create_signed_installer") {
   ]
   outputs = [
     "$root_out_dir/brave_installer.exe",
+    "$root_out_dir/$brave_installer_exe",

```

Comment:
I didn't delete `"$root_out_dir/brave_installer.exe",` to preserve existing outputs.
I think this can be removed after CI doesn't use `brave_installer.exe` anymore.